### PR TITLE
Update URL format for sitemap redirect

### LINKED ISSUE #61: Define Routing
- [x] Utilize folder mapping to define paths where product pages are used
    - Feature currently undocumented, but can be seen in https://github.com/hlxsites/worldbank-wbl/blob/main/fstab.yaml
    - In this example, any requests to resources below `/data/exploreeconomics` are redirected to a document named `default` at `/data/exploreeconomics/default`
    - Open issue that might prevent us from fully using it: https://github.com/adobe/helix-html-pipeline/issues/30
- [x] Add redirects so that old paths still work. Maidenform uses mostly paths with `.html` suffix.
    - [X] Added node.js tool which extracts old URLs from `sitemap.xml` and generates a `redirects.xlsx`
- [x] Update implementation to new product url format (e.g. `/products/{url-key}/{sku}`).
    - [x] Sitemap redirect
    - [x] Metadata
    - [x] Sitemap

### PR DESCRIPTION
**Pull request by herzog31**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #61

Test URLs:
- Before: https://main--maidenform--hlxsites.hlx.page/
- After: https://bug-sitemap-redirect--maidenform--hlxsites.hlx.page/

### CODE REVIEW

File: tools/sitemap-redirect/sitemap-redirect.js [hunk 1]
```diff
@@ -15,35 +42,59 @@ const url = 'https://www.maidenform.com/sitemap.xml';
...
+    // Go through all URLs
+    for(let url of urls) {
+        // Use new PDP format if URL points to product
+        const matchingProduct = allProducts.find(({ sku, url_key }) => 
+            url.indexOf(`${url_key}.html`) > -1);
+
+        if (matchingProduct) {
+            const { sku, url_key } = matchingProduct;
+            data.push([url, `/products/${url_key}/${sku}`])
+            continue;;
```
Comment (hannessolo) [inline tools/sitemap-redirect/sitemap-redirect.js:91]: one of the `;` should probably be on the line before

**Review by hannessolo — APPROVED** [review]
Looks good except for one minor change.

TBH I don't 100% understand what this is for. We run this script manually and then upload the redirects file, to get redirects from `.html`?

**Comment by herzog31** [comment]
This is for making sure the existing URLs from the old pages (e.g. from google) will still work.

**Comment by hannessolo** [comment]
👍 merge when ready

**Pull request merged**